components/camel-as2/camel-as2-component/src/test/resources/mendelson/key4.cer

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file should be provided via parameters instead of being kept in the code base (i.e.: `mvn -Das2.http.key=/path/to/key`).

Reply:
That's a bad practice. Other reviewer may have missed.